docs/developer-docs/integrations/ledger/ledger-local-setup.md

```diff
@@ -11,27 +11,27 @@ If you don’t have the IC SDK installed, follow instructions on the [installing
 ### Step 2: Create a new dfx project with the command:
 
 ```
-dfx new ledger
-cd ledger
+dfx new ledger-canister
+cd ledger-canister
 ```
 
 ### Step 3:  Determine ledger file locations
 
 Go to the [releases overview](https://dashboard.internetcomputer.org/releases) and copy the latest replica binary revision. At the time of writing, this is `a17247bd86c7aa4e87742bf74d108614580f216d`.
 
-The URL for the ledger WASM module is `https://download.dfinity.systems/ic/<REVISION>/canisters/ic-icrc1-ledger.wasm.gz`, so with the above revision it would be `https://download.dfinity.systems/ic/a17247bd86c7aa4e87742bf74d108614580f216d/canisters/ic-icrc1-ledger.wasm.gz`.
+The URL for the ledger Wasm module is `hhttps://download.dfinity.systems/ic/<REVISION>/canisters/ledger-canister.wasm.gz`, so with the above revision it would be `https://download.dfinity.systems/ic/a17247bd86c7aa4e87742bf74d108614580f216d/canisters/ledger-canister.wasm.gz`.
```

Comment:
```suggestion
The URL for the ledger Wasm module is `https://download.dfinity.systems/ic/<REVISION>/canisters/ledger-canister.wasm.gz`, so with the above revision it would be `https://download.dfinity.systems/ic/a17247bd86c7aa4e87742bf74d108614580f216d/canisters/ledger-canister.wasm.gz`.
```